examples/eve/lib/utils.ts

```
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from "clsx";
```

ESLint

[error] Extra semicolon.

(semi)